### PR TITLE
Added new preview version for container instances

### PR DESCRIPTION
**Pull request by dobrebogdan**
Creating a new preview api version with a new property "identityAcls" and related classes for container instances.

### CODE REVIEW

**Comment by dobrebogdan** [comment]
The Swagger Avocado check threw an error before due to not having the latest version in the API, but after updating with the latest version now the check throws some seemingly unrelated errors (about an old version). Is this check broken?

**Comment by dobrebogdan** [comment]
Not using the latest api version as default due to some missing API paths compared to 2024-05 that trigger this error for the Avocado check: [MISSING_APIS_IN_DEFAULT_TAG](https://github.com/Azure/avocado/blob/main/README.md#missing_apis_in_default_tag)

File: specification/containerinstance/resource-manager/Microsoft.ContainerInstance/preview/2024-11-01-preview/containerInstance.json [hunk 1]
```diff
@@ -0,0 +1,2663 @@
...
+          ],
+          "x-ms-enum": {
+            "name": "IdentityAccessLevel",
+            "modelAsString": true
+          }
+        },
+        "acls": {
+          "description": "The access control levels for each identity.",
+          "type": "array",
+          "items": {
```
Comment (raosuhas) [inline specification/containerinstance/resource-manager/Microsoft.ContainerInstance/preview/2024-11-01-preview/containerInstance.json:2567]: plese check the swagger lint diff warning for this property : 

[https://github.com/Azure/azure-rest-api-specs/pull/30879/checks?check_run_id=31300522952](https://github.com/Azure/azure-rest-api-specs/pull/30879/checks?check_run_id=31300522952)

[https://github.com/Azure/azure-openapi-validator/blob/main/docs/xms-identifier-validation.md](https://github.com/Azure/azure-openapi-validator/blob/main/docs/xms-identifier-validation.md)

Comment (dobrebogdan) [reply specification/containerinstance/resource-manager/Microsoft.ContainerInstance/preview/2024-11-01-preview/containerInstance.json:2567]: Thank you, I fixed that issue, please let me know if there is anything else to fix.

Comment (dobrebogdan) [reply specification/containerinstance/resource-manager/Microsoft.ContainerInstance/preview/2024-11-01-preview/containerInstance.json:2567]: Hello @raosuhas , is there any update on this PR? Thanks!

**Pull request closed without merging**